framework/web/Request.php

```diff
@@ -234,6 +234,9 @@ class Request extends \yii\base\Request
      */
     public $ipHeaders = [
         'X-Forwarded-For',
+        'cf-connecting-ip',
```

Comment:
can you give a source for this one, why do you think it is relevant?

Reply:
It's for Cloudflare https://support.cloudflare.com/hc/en-us/articles/200170986-How-does-Cloudflare-handle-HTTP-Request-headers-

Reply:
As I read that page, `cf-connecting-ip` should be preferred over `X-Forwarded-For` because cloudflare will not override an existing `X-Forwarded-For` header but rather just append the value of `cf-connecting-ip`. That means `cf-connecting-ip` should be added before `X-Forwarded-For` in this array. Also `cf-connecting-ip` must be added to the list of `$secureHeaders` property.

Reply:
maybe yes, I just want to show another ways how to get real user IP